src/storage/copier/var_list_column_chunk.cpp

```diff
@@ -90,10 +90,16 @@ void VarListColumnChunk::write(const common::Value& listVal, uint64_t posToWrite
 
 void VarListColumnChunk::setValueFromString(const char* value, uint64_t length, uint64_t pos) {
     auto listVal =
-        TableCopyUtils::getArrowVarList(value, 1, length - 2, dataType, *copyDescription);
+        TableCopyUtils::getVarListValue(value, 1, length - 2, dataType, *copyDescription);
     write(*listVal, pos);
 }
 
+void VarListColumnChunk::resetToEmpty() {
+    ColumnChunk::resetToEmpty();
+    varListDataColumnChunk.dataChunk->resetToEmpty();
```

Comment:
I would move these two lines for `varListDataColumnChunk` to `VarListDataColumnChunk::reset()`